docker-compose.yml

```diff
@@ -73,7 +75,33 @@ services:
       --validator-keys="/charon/node1:/charon/node1"
       --validators-proposer-default-fee-recipient="0x0000000000000000000000000000000000000000"
     volumes:
-     - .:/charon
+      - .:/charon
+
+  vc2-teku:
```

Comment:
maybe extract common teku config as `x-teku-base` and then use it as a common base for the teky VCs `<<: *teku-base`?

Reply:
Although, I'm not sure how to do this for common command line flags... it seems `command` needs to be duplicated every time... so not sure it is worth it...

Reply:
command line flags needs to know the node for beacon node endpoint